### PR TITLE
fix: Improve documentation

### PR DESCRIPTION
**Pull request by Rhuagh**
This change is [Reviewable](https://reviewable.io/reviews/omni-viral/gfx_mem/3)

### CODE REVIEW

File: src/block.rs [hunk 1]
```diff
@@ -20,7 +20,15 @@ pub trait Block<B: Backend> {
...
-    /// Helper method to check if `other` block is sub-block of `self`
+    /// Check if a block is a child of this block.
+    ///
+    /// ### Parameters:
+    ///
+    /// - `other`: potential child block
+    ///
+    /// ### Type parameters:
+    ///
+    /// - `Y`: tag of potential child block
```
Comment (zakarumych) [inline src/block.rs]: `T`: type of potential child block

**Comment by zakarumych** [comment]
I've rebased this.

**Review by zakarumych — CHANGES_REQUESTED** [review]
Thank you! I really appreciate you help with documentation!

**Comment by Rhuagh** [comment]
Fixed.

**Review by zakarumych — APPROVED** [review]
Great! Thanks!

**Pull request merged**